kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_multiple_user_tasks_with_complex_variables_process.bpmn

```diff
@@ -3,23 +3,23 @@
   <bpmn:process id="simple_multiple_user_tasks_with_complex_variables_process" isExecutable="true">
     <bpmn:extensionElements>
       <camunda:properties>
-        <camunda:property name="taskana-attributes" value="attribute1,attribute2,attribute3" />
-        <camunda:property name="taskana.domain" value="DOMAIN_A" />
+        <camunda:property name="kadai-attributes" value="attribute1,attribute2,attribute3" />
+        <camunda:property name="kadai.domain" value="DOMAIN_A" />
       </camunda:properties>
     </bpmn:extensionElements>
     <bpmn:startEvent id="StartEvent_0c8j229">
       <bpmn:outgoing>SequenceFlow_182avfz</bpmn:outgoing>
     </bpmn:startEvent>
     <bpmn:sequenceFlow id="SequenceFlow_182avfz" sourceRef="StartEvent_0c8j229" targetRef="Task_0k9mjh4" />
-    <bpmn:serviceTask id="Task_0k9mjh4" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="Task_0k9mjh4" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```

Comment:
io.kadai.impl.ComplexProcessVariableSetter ?

Reply:
You're right, I will change it